Webpack error in browser should include the useful information

If you are missing a files from `node_modules`, which seems to happen way more often than you would like you get a quite needlessly filtered error output in the browser that does not say much about the problem.

If you do e.g.
```
npx @vaadin/cli init err
cd err
mvn
```
ctrl-c when server is started
```
rm node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout-styles.js
mvn
```

What you will see in the browser when opening the app is
```
Webpack Error
Webpack build failed with errors:
ERROR in ../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout.js
ERROR in ../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout.js
```

Problems here are:
1. There is no ERROR in the given file, the problem is that it is missing
2. Based on the error message, there is nothing I can do to try to fix it except for `mvn clean`, which does not help
3. The more useful information is output to the server console, but hidden way up in the log so you won't necessarily notice the error by looking at the log in e.g. Eclipse 
```
Running webpack to compile frontend resources. This may take a moment, please stand by...
ERROR in ../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout.js
Module not found: Error: Can't resolve './vaadin-horizontal-layout-styles.js' in '/.../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo'
 @ ../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout.js 1:0-46
 @ ../target/frontend/generated-flow-imports.js

ERROR in ../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout.js
Module not found: Error: Can't resolve './vaadin-horizontal-layout-styles.js' in '.../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo'
 @ ../node_modules/@vaadin/vaadin-ordered-layout/theme/lumo/vaadin-horizontal-layout.js 1:0-46
 @ ../target/frontend/generated-flow-imports.js
ℹ ｢wdm｣: Failed to compile.
```
While this message is not super useful either it still makes me realize that the problem is a missing file in `node_modules` so the problem is not in my project but in a dependency or the installation of the dependencies.

The relevant info should definitely not be removed. I think it used to be there in earlier versions. Tested with 14.1.20

--- a/flow-server/src/main/java/com/vaadin/flow/server/DevModeHandler.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/DevModeHandler.java
@@ -481,7 +481,6 @@ public final class DevModeHandler {
         Consumer<String> warn = s -> getLogger()
                 .debug(String.format(YELLOW, "{}"), s);
         Consumer<String> log = info;
-        boolean storeOutput = false;
         for (String line; ((line = reader.readLine()) != null);) {
             String cleanLine = line
                     // remove color escape codes for console
@@ -490,22 +489,13 @@ public final class DevModeHandler {
                     .replaceAll("\\?babel-target=[\\w\\d]+", "");
 
             // write each line read to logger, but selecting its correct level
-            log = line.contains("WARNING") ? warn
-                    : line.contains("ERROR") ? error : log;
+            log = line.contains("WARNING") ? warn :
+                    line.contains("ERROR") ? error :
+                      isInfo(line, cleanLine) ? info : log;
             log.accept(cleanLine);
 
-            if (!storeOutput && log.equals(error)) {
-                // Begin storing output lines when encountering an error.
-                storeOutput = true;
-            } else if (storeOutput && (line.trim().isEmpty() || cleanLine.trim()
-                    .startsWith("i"))) {
-                // Empty line breaks error block or a 'i' marking for info
-                storeOutput = false;
-                output.append(System.lineSeparator());
-            }
-
             // Only store webpack errors to be shown in the browser.
-            if (storeOutput) {
+            if (log.equals(error)) {
                 // save output so as it can be used to alert user in browser.
                 output.append(cleanLine).append(System.lineSeparator());
             }
@@ -519,12 +509,15 @@ public final class DevModeHandler {
                 failedOutput = failed ? output.toString() : null;
                 // reset output and logger for the next compilation
                 output = getOutputBuilder();
-                storeOutput = false;
                 log = info;
                 // Notify DevModeHandler to continue
                 doNotify();
             }
         }
+    }
+
+    private boolean isInfo(String line, String cleanLine) {
+        return line.trim().isEmpty() || cleanLine.trim().startsWith("i");
     }
 
     private StringBuilder getOutputBuilder() {

--- a/flow-server/src/main/java/com/vaadin/flow/server/DevModeHandler.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/DevModeHandler.java
@@ -480,6 +480,7 @@ public final class DevModeHandler {
         Consumer<String> warn = s -> getLogger()
                 .debug(String.format(YELLOW, "{}"), s);
         Consumer<String> log = info;
+        boolean storeOutput = false;
         for (String line; ((line = reader.readLine()) != null);) {
             String cleanLine = line
                     // remove color escape codes for console
@@ -492,8 +493,18 @@ public final class DevModeHandler {
                     : line.contains("ERROR") ? error : log;
             log.accept(cleanLine);
 
+            if (!storeOutput && log.equals(error)) {
+                // Begin storing output lines when encountering an error.
+                storeOutput = true;
+            } else if (storeOutput && (line.trim().isEmpty() || cleanLine.trim()
+                    .startsWith("i"))) {
+                // Empty line breaks error block or a 'i' marking for info
+                storeOutput = false;
+                output.append(System.lineSeparator());
+            }
+
             // Only store webpack errors to be shown in the browser.
-            if (line.contains("ERROR")) {
+            if (storeOutput) {
                 // save output so as it can be used to alert user in browser.
                 output.append(cleanLine).append(System.lineSeparator());
             }
@@ -507,6 +518,7 @@ public final class DevModeHandler {
                 failedOutput = failed ? output.toString() : null;
                 // reset output and logger for the next compilation
                 output = getOutputBuilder();
+                storeOutput = false;
                 log = info;
                 // Notify DevModeHandler to continue
                 doNotify();

